apps/client/app/features/artboard/bucketFill.ts

```diff
@@ -0,0 +1,167 @@
+// ref: https://github.com/cat-crosswalk/nascalay-frontend/blob/main/src/components/Canvas/bucketFill.ts
+
+import type { Color, LikeEqualColor } from "./types";
+import { colorsToRaw, equalColor, hexToColor, rawToColors } from "./utils";
+
+// https://nullpon.moe/dev/sample/canvas/bucketfill.html めちゃめちゃ参考にしてる
+// シードフィルアルゴリズムで塗りつぶす
+
+/**
+ * 指定された座標から右方向にまっすぐ targetColor と等しい色を塗りつぶす
+ *
+ * @returns 右端の x 座標
+ */
+function drawToRight(
+  data: Color[][],
```

Comment:
```suggestion
  colorMap: Color[][],
```
とか`pixelData`とかだと苦なく読める

Reply:
PR 出してるし前言ったけど bucketFill は修正しないね

向こうでも data 使ってるからそっちに書いてくれ